### PR TITLE
[CLI] export archive and unarchive a sketch

### PR DESCRIPTION
**Pull request by jaegeral**
Adding CLI capabilities to `archive` `unarchive` and `export` a sketch.

### CODE REVIEW

File: cli_client/python/timesketch_cli_client/commands/sketch.py [hunk 1]
```diff
@@ -93,3 +95,67 @@ def create_sketch(ctx, name, description):
...
+    click.echo("Executing export . . . ")
+    click.echo("Depending on the sketch size, this can take a while")
+    # start counting the time the export took
+    start_time = time.time()
+    try:
+        search_obj = search.Search(sketch=sketch)
+
+        click.echo(f"Number of events in that sketch: {search_obj.expected_size}")
+
+        search_obj.to_file(filename)
```
Comment (jkppr) [inline cli_client/python/timesketch_cli_client/commands/sketch.py:119]: You are only catching `ValueError` but what about e.g. `IOError` if there are issues with writing the file?

File: cli_client/python/timesketch_cli_client/commands/sketch.py [hunk 2]
```diff
@@ -93,3 +95,67 @@ def create_sketch(ctx, name, description):
...
+        # Using the sketch.export function might also work
+        # TODO: https://github.com/google/timesketch/issues/2344
+        # sketch.export("./test.zip")
```
Comment (jkppr) [inline cli_client/python/timesketch_cli_client/commands/sketch.py:122]: Does it work or not? Maybe we should be a bit more clear on the message here :)

**Review by jkppr — APPROVED** [review]
lgtm - some minor nits below

File: cli_client/python/timesketch_cli_client/commands/sketch.py (hunk 2 above)
Comment (jaegeral) [reply cli_client/python/timesketch_cli_client/commands/sketch.py:122]: I did some tests and with more than 10 mil events on a local system it timed out, but this is not related to the CLI client

File: cli_client/python/timesketch_cli_client/commands/sketch.py [hunk 3]
```diff
@@ -93,3 +95,67 @@ def create_sketch(ctx, name, description):
...
+        end_time = time.time()
+        click.echo(f"Export took {end_time - start_time} seconds")
+        click.echo("Finish")
+    except ValueError as e:
+        click.echo(f"Error: {e} {e.message}")
```
[the remark this answers is not in the document]
Comment (jaegeral) [reply cli_client/python/timesketch_cli_client/commands/sketch.py:127]: true

File: cli_client/python/timesketch_cli_client/commands/sketch.py (hunk 1 above)
Comment (jaegeral) [reply cli_client/python/timesketch_cli_client/commands/sketch.py:119]: but I think I will keep it :-)

**Pull request merged**